apps/studio/components/interfaces/Settings/Infrastructure/InfrastructureInfo.tsx

```diff
@@ -57,10 +57,12 @@ const InfrastructureInfo = () => {
   return (
     <>
       <ScaffoldDivider />
-
-      <InstanceConfiguration />
-
-      <ScaffoldDivider />
+      {project?.cloud_provider !== 'FLY' && (
```

Comment:
If you open this page before project loads there is a brief period where it flashes into existence, but you pretty much have to land on the page immediately for that to happen